docs/spec_draft.md

```diff
@@ -82,13 +82,70 @@ The specification of an op comprises of the following components (in the order
   * **Constraints** Type constraints on the operand(s), result(s).
   * **Examples** Examples demonstrating the working of the op.
 
+### stablehlo.abs
+
+`stablehlo.abs(operand) -> result`
+
+### Semantics
+
+Performs element-wise absolute value of `operand` tensor and produces a `result`
+tensor. For a complex value `(a, b)`, the absolute value is defined as
+$\sqrt(a^2 + b^2)$.
+
+For n-bit signed integer, the absolute value of $-2^{n-1}$ is implementation
+defined and one of the followings:
+
+  * Saturation to $2^{n-1}-1$
+  * $-2^n-1$
+
```

Comment:
What about special cases for floating point? (Same question for `negate`).

Reply:
added.